plexapi/__init__.py

```diff
@@ -17,6 +17,7 @@
 VERSION = '3.0.6'
 TIMEOUT = CONFIG.get('plexapi.timeout', 30, int)
 X_PLEX_CONTAINER_SIZE = CONFIG.get('plexapi.container_size', 100, int)
+X_PLEX_ENABLE_FAST_CONNECT = CONFIG.get('plexapi.enable_fast_connect', False, bool)
```

Comment:
Can you please update the configuration.rst file with what X_PLEX_ENABLE_FAST_CONNECT is and when we would want to use it.

https://github.com/pkkid/python-plexapi/blob/master/docs/configuration.rst